mqtt/mqtt-edgehub/src/auth/authentication/edgehub.rs

```diff
@@ -69,7 +72,17 @@ impl Authenticator for EdgeHubAuthenticator {
         &self,
         context: AuthenticationContext,
     ) -> Result<Option<AuthId>, Self::Error> {
-        let auth_id = self.authenticate(context).await?;
+        let auth_id = (|| async {
+            info!("authenticate client");
+            let auth_id = self.authenticate(&context).await?;
+            Ok(auth_id)
+        })
+        .retry(ExponentialBackoff {
+            max_elapsed_time: Some(Duration::from_secs(60)),
+            ..ExponentialBackoff::default()
+        })
+        .await?;
```

Comment:
For reference:
https://docs.rs/backoff/0.2.1/backoff/exponential/struct.ExponentialBackoff.html